BO4E/BO/Rechnung.cs

```diff
@@ -503,5 +503,20 @@ private DateTime _Faelligkeitsdatum
         [FieldName("invoiceItemList", Language.EN)]
         public List<Rechnungsposition> Rechnungspositionen { get; set; }
 
+        /// <summary>
+        ///     Kennzeichen, ob es sich um eine selbstausgestellte Rechnung handelt
+        /// </summary>
+        [ProtoMember(23)]
+        [JsonProperty(Required = Required.Default, Order = 23, PropertyName = "selbstausgestellt")]
+        [JsonPropertyName("selbstausgestellt")]
```

Comment:
die namen bitte noch entsprechend anpassen:
```suggestion
        [JsonProperty(Required = Required.Default, Order = 23, PropertyName = "istSelbstausgestellt")]
        [JsonPropertyName("istSelbstausgestellt")]
```